hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/om/TestScmChillMode.java

```diff
@@ -158,25 +154,21 @@ public void testChillModeOperations() throws Exception {
 
     cluster.stop();
```

Comment:
Let's move the `cluster.stop()` call to an `@After` method. Actually it looks like you already have cluster.shutdown(). Why do we need a stop call here?